pkg/cvo/cvo.go

```diff
@@ -674,7 +674,6 @@ func (optr *Operator) getOrCreateClusterVersion(ctx context.Context, enableDefau
 			Name: optr.name,
 		},
 		Spec: configv1.ClusterVersionSpec{
-			Upstream:  upstream,
 			Channel:   "fast",
```

Comment:
Why can't we remove the fast channel as default?

Reply:
It's good to have a signal that this code is happening, because it's very serious admin errors or a bug when it trips.  For example, we had [rhbz#1708697](https://bugzilla.redhat.com/show_bug.cgi?id=1708697), from the CVO racing cluster-bootstrap during install (fixed by telling the CVO not do push this default into the cluster during install).

Reply:
I can live with that.